src/rust/src/properties_reader.rs

```diff
@@ -9,19 +9,21 @@ use std::str;
 
 /// Access to current feature
 pub struct FgbFeature {
-    pub(crate) header_buf: Vec<u8>,
+    pub(crate) header_buf: Vec<u8>, // Header would require lifetime
```

Comment:
Can you clarify this comment?

Reply:
Would `Type "Header" would require lifetime for struct` be understandable?

Reply:
Sorry, I'm still not sure what the comment is trying to tell me. 


Is it that you are trying to document that `header()` borrows this `header_buf` field?

```
let some_fgb_feature: FgbFeature = get_fgb_feature();

// __this call borrows `header_buf` with lifetime `a __
let header: pub struct Header<'a> = some_fgb_feature.header();
```

Reply:
Ah, I think I get it.

You're saying that you chose for FgbFeature to have a `header_buf: Vec<u8>` field rather than a `header: Header` field because having a `Header` field would require FgbFeature to cary a generic lifetime, like FgbFeature<'a>. 

Is that it?